src/main/java/org/folio/dew/batch/acquisitions/edifact/services/SaveToFileStorageService.java

```diff
@@ -0,0 +1,57 @@
+package org.folio.dew.batch.acquisitions.edifact.services;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.folio.dew.domain.dto.EdiFtp;
+import org.folio.dew.domain.dto.VendorEdiOrdersExportConfig;
+import org.folio.dew.repository.FTPObjectStorageRepository;
+import org.folio.dew.repository.SFTPObjectStorageRepository;
+import org.springframework.stereotype.Service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import lombok.extern.log4j.Log4j2;
+
+@RequiredArgsConstructor
+@Service
+@Log4j2
+public class SaveToFileStorageService {
```

Comment:
I think this class should be renamed, because File Storage associated with Minio for me, not with FTP